guides/operations/setup/installing_sqream_with_monit.rst

```diff
@@ -103,10 +105,559 @@ Note the following:
+
+For example, you can stop the **sqream3** service (being monitored by Monit) in one of the following ways:
+
+* Stopping Monit and SQream separately.
+* Stopping SQream using a Monit command.
+
+Both examples above are described in more detail below.
+
+Stopping Monit and SQream Separately
+-------------------------------------
+
+You can stop the Monit service and SQream separately as follows:
+
+.. code-block:: console
+     
+   $ sudo systemctl stop monit
+   $ sudo systemctl stop sqream3
+
+You can restart Monit as follows:
+
+.. code-block:: console
+
+   $ sudo systemctl start monit
+      
+Restarting Monit automatically restarts the SQream services.      
+
+Stopping SQream Using a Monit Command
+-------------------------------------
+
+You can stop SQream using a Monit command as follows:
+
+.. code-block:: console
+     
+   $ sudo monit stop sqream3
+
+This command spots SQream only (and not Monit).
+
+You can restart SQream as follows:
+
+.. code-block:: console
+     
+   $ sudo monit start sqream3
+       
+Monit Command Line Options
+-------------------------------------
+This section describes some of the most commonly used Monit command options:
+
+.. code-block:: console
+     
+   $ start all             - Start all services
+   $ start <name>          - Only start the named service
+   $ stop all              - Stop all services
+   $ stop <name>           - Stop the named service
+   $ restart all           - Stop and start all services
+   $ restart <name>        - Only restart the named service
+   $ monitor all           - Enable monitoring of all services
+   $ monitor <name>        - Only enable monitoring of the named service
+   $ unmonitor all         - Disable monitoring of all services
+   $ unmonitor <name>      - Only disable monitoring of the named service
+   $ reload                - Reinitialize monit
+   $ status [name]         - Print full status information for service(s)
+   $ summary [name]        - Print short status information for service(s)
+   $ report [up|down|..]   - Report state of services. See manual for options
+   $ quit                  - Kill the monit daemon process
+   $ validate              - Check all services and start if not running
+   $ procmatch <pattern>   - Test process matching pattern
+
+Using Monit to Upgrade your Version of SQream
+=============================================
+
+You can use Monit to upgrade your version of SQream.
+
+**To use Monit to upgrade your version of SQream:**
+
+1. Stop all actively running SQream services:
+
+   .. code-block:: console
+     
+      $ sudo monit stop all
+      
+2. Verify that SQream has stopped listening on ports **500X**, **510X**, and **310X**:
+
+   .. code-block:: console
+
+      $ sudo netstat -nltp    #to make sure sqream stopped listening on 500X, 510X and 310X ports.
+
+3. Stop the UI (not as a super user, but as a SQream user):
```

Comment:
The UI (Studio) may not be even installed. In any case, having it running will cause any issues so I think this line should be removed. Furthermore, the UI may be installed not as pm2 but as a Docker container.

Reply:
The UI (Studio) may not be even installed - **Then it should be an optional step.**

In any case, having it running will cause any issues so I think this line should be removed - **It may already be running, which is why the step says to stop it.**

Furthermore, the UI may be installed not as pm2 but as a Docker container - **in that case, what is the correct command?**

Reply:
You can remove the entire line. Less confusion

Reply:
> You can remove the entire line. Less confusion

Step removed.